src/ts/components/subtitleoverlay.ts

```diff
@@ -85,6 +85,13 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
+        domElement.css('left', `${event.position.column}em`);
```

Comment:
Ok, I agree that there is incompatibility and this solution is far from perfect. 

To properly separate the two we would need this info before, since from the UI we only know if CEA-608 happen when the caption arrives.

There is more context and discussion in the related issue.